impl_core/src/test/java/io/opencensus/implcore/trace/propagation/B3FormatTest.java

```diff
@@ -217,5 +218,8 @@ public void fields_list() {
     assertThat(b3Format.fields())
         .containsExactly(
             X_B3_TRACE_ID, X_B3_SPAN_ID, X_B3_PARENT_SPAN_ID, X_B3_SAMPLED, X_B3_FLAGS);
+    for (String header : b3Format.fields()) {
+      HttpUtils.assertHeaderNameIsValid(header);
```

Comment:
It's great that we are testing that `X_B3*` are valid, however, it might suffice for us to get the originally offending header and add that to assert a failure, thus preventing any future regressions.